### PR TITLE
Update docs for adding custom cloud credentials

### PR DESCRIPTION
**Pull request by kaushikb11**

### CODE REVIEW

File: platform/upgrades/adding-custom-cloud-credentials.md [hunk 1]
```diff
@@ -80,21 +80,21 @@ The final step is to add all the Grid policies to your account. That means that
...
+For the next step, you're going to create a role using terraform. Make sure you have `git`, `terraform`, `jq`, and `AWS CLI` installed on your machine. Installation instruction of these tools are [available](adding-custom-cloud-credentials.md#installing-3rd-party-tools). If you're familiar with terraform we recommend you check the terraform module we'll be using to create necessary roles & policies. [https://github.com/gridai/terraform-aws-gridbyoc](https://github.com/gridai/terraform-aws-gridbyoc) This module is published on the official terraform registry for your convenience [https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest](https://registry.terraform.io/modules/gridai/gridbyoc/aws/latest)
```
Comment (nmiculinic) [inline platform/upgrades/adding-custom-cloud-credentials.md:94]: Expand this a bit what's the significance of this role

Comment (kaushikb11) [reply platform/upgrades/adding-custom-cloud-credentials.md:94]: Should we mention AWS ARN role?

Comment (nmiculinic) [reply platform/upgrades/adding-custom-cloud-credentials.md:94]: ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Comment (nmiculinic) [reply platform/upgrades/adding-custom-cloud-credentials.md:94]: ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Comment (nmiculinic) [reply platform/upgrades/adding-custom-cloud-credentials.md:94]: ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to.

Comment (nmiculinic) [reply platform/upgrades/adding-custom-cloud-credentials.md:94]: ARN is AWS resource Name or specific resource, that is the AWS Role we're referring to. Not sure what you're referring to by mentioning it.

**Comment by zippeurfou** [comment]
@kaushikb11 @nmiculinic any update on this one?

**Comment by zippeurfou** [comment]
@nmiculinic @kaushikb11 is it still relevant or should we close?

**Comment by essiequoi** [comment]
Closing this and reopening if needed at later time

**Pull request closed without merging**